src/python/library/tritonclient/http/_utils.py

```diff
@@ -35,16 +35,15 @@ def _get_error(response):
     indicates the error. If no error then return None
     """
     if response.status_code != 200:
-        body = response.read()
+        body = response.read().decode()
         try:
             error_response = json.loads(body) if len(body) else {"error": ""}
             return InferenceServerException(msg=error_response["error"],
-                                            status=str(response.status_code),
-                                            debug_details=body.decode())
+                                            status=str(response.status_code))
         except json.JSONDecodeError as e:
-            return InferenceServerException(msg=str(e),
+            return InferenceServerException(msg="",
```

Comment:
Let's change the `msg` to "An exception occurred in the client while decoding the JSON. error: e". For debug details we can pass body.